cmd/k8s-operator/deploy/chart/values.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Tailscale Inc & AUTHORS
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Operator oauth credentials. If set a Kubernetes Secret with the provided
+# values will be created in the operator namespace. If unset a Secret named
+# operator-oauth must be precreated.
+# oauth:
+#   clientId: ""
+#   clientSecret: ""
+
+operatorConfig:
+  image:
+    repo: tailscale/k8s-operator
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+  logging: "info"
+  hostname: "tailscale-operator"
+  nodeSelector:
+    kubernetes.io/os: linux
+  resources: {}
```

Comment:
I took out the resource request that we had. I think it does not make sense to have these values here- the actual requests will depend on the amount and size of resources that the operator needs to parse, so users would  have to figure it out for themselves anyway. These requests were a lot higher than a user who just wants to set up a couple proxies would need and were making the deployment to fail on clusters that don't have a large amount of  free resources.
For reference bitnami charts, also don't preset resource requests/limits on their charts https://github.com/bitnami/charts/blob/main/bitnami/cert-manager/values.yaml#L129-L131